src/Engine/Graph/Classifier.cpp

```diff
@@ -36,7 +37,7 @@ Classifier::Classifier(Graph* parentNode) : Graph(parentNode)
 	SetName("Classifier");
 	
 	mIsDirty		= false;
-	mIsRunning		= true;
+	mIsRunning		= false;
```

Comment:
I really worry about this low-level change in Engine's Classifier.
It is used at other places, for instance the EngineJNI and the Apps...
Can you elaborate, why this is necessary?
How does this affect the Classifier while NOT in a running session but e.g. in Design mode?
My concerns are also about the new Start() and Stop() mechanisms

Reply:
Well, this seemed logical in the context of file writing. When there is no running session, the Classifier anyway stays running by default. So this change is done to make it Running with session. 
If this may break something else, I'll try to keep this parts without modification and solve this other way. 
Basically, we don't need to write to a file when a session is not running.

Reply:
@narek-h BTW: The code that I worry about regarding this change is the C API of the Engine which you find in:
https://github.com/neuromore/studio/blob/master/src/Engine/neuromoreEngine.cpp
https://github.com/neuromore/studio/blob/master/src/Engine/neuromoreEngine.h
We use this in the Apps (directly in iOS and using a JNI Wrapper on Android).
Maybe you could have some quick review on that file and check if and how this change affects it?